src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -61,7 +61,6 @@ public sealed partial class TextLoader
                 var readerEst = StaticPipeUtils.ReaderEstimatorAnalyzerHelper(env, ch, ctx, rec, func);
                 Contracts.AssertValue(readerEst);
                 var reader = readerEst.Fit(files);
```

Comment:
>var reader = readerEst.Fit(files); [](start = 16, length = 34)

FYI if you happen to post another iteration, you may as well clean this up too.

Reply:
Not worth posting a new iteration though... just only if you have to for some other reason anyway.